keyper/__init__.py

```diff
@@ -330,25 +331,24 @@ def list_keychains(*, domain=None):
             log.error("Failed to get keychains: %s", ex)
             raise
 
-        keychains = keychains.split("\n")
-
         # Cleanup the output format into a regular Python list of strings
-        keychains = [keychain.strip() for keychain in keychains]
-        keychains = [keychain for keychain in keychains if len(keychain) > 0]
-        keychains = [keychain[1:] for keychain in keychains]
-        keychains = [keychain[:-1] for keychain in keychains]
+        result = []
```

Comment:
Can we use something a little more descriptive than `result` here? Perhaps use `keychains` and rename the previous variable to something like `keychain_command_output`?

Reply:
Ya, I went back and forth with a few names but just threw in `result` as a bit of a placeholder so thank you for the feedback! I will update the naming right away.